### ISSUE TITLE
Investigate lan-mouse

### ISSUE THREAD

**Issue by castrojo** [ublue-os/bluefin#1665]
Didn't know this existed but looks pretty cool, we should compare with input-leap: https://github.com/feschber/lan-mouse

**Comment by castrojo** [comment]
Anyone have any updated opinions on this?

**Comment by tulilirockz** [comment]
I dont think this one is necessary anymore since the Input Leap flatpak exists now :)
